org.freedesktop.Sdk.Extension.ldc2.yml

```diff
@@ -0,0 +1,49 @@
+app-id: org.freedesktop.Sdk.Extension.ldc2
+branch: '19.08'
+runtime: org.freedesktop.Sdk
+runtime-version: '19.08'
+sdk: org.freedesktop.Sdk
+build-extension: true
+separate-locales: false
+appstream-compose: false
+build-options:
+  prefix: /usr/lib/sdk/ldc2
+modules:
+  - name: ldc2
+    buildsystem: simple
+    build-commands:
+      - mkdir -p /usr/lib/sdk/ldc2
+      - mv bin etc import lib lib32 /usr/lib/sdk/ldc2
+    sources:
+      - type: archive
+        only-arches:
+          - x86_64
+        url: https://github.com/ldc-developers/ldc/releases/download/v1.20.0/ldc2-1.20.0-linux-x86_64.tar.xz
+        sha256: ab2100228b9396ff1098006f7692d5638f4ebeb07890767499207c8aaf62ff09
+      - type: archive
+        only-arches:
+          - aarch64
+        url: https://github.com/ldc-developers/ldc/releases/download/v1.20.0/ldc2-1.20.0-linux-aarch64.tar.xz
+        sha256: 3b45670b11507fc613661f2fa9bb90611d6a75d8f2b422d2d76788f456a51295
+
+  - name: scripts
+    buildsystem: simple
+    build-commands:
+      - 'cp enable.sh /usr/lib/sdk/ldc2'
+    sources:
+      - type: script
+        commands:
+          - 'export PATH="$PATH:/usr/lib/sdk/ldc2/bin"'
+          - 'export LIBRARY_PATH="$LIBRARY_PATH:/usr/lib/sdk/ldc2/lib"'
+          - 'export LD_LIBRARY_PATH="LD_LIBRARY_PATH:/usr/lib/sdk/ldc2/lib"'
+        dest-filename: enable.sh
+
+  - name: appdata
+    buildsystem: simple
+    build-commands:
+      - mkdir -p ${FLATPAK_DEST}/share/appdata
+      - cp org.freedesktop.Sdk.Extension.ldc2.appdata.xml ${FLATPAK_DEST}/share/appdata
```

Comment:
New appdata files should use metainfo instead. https://www.freedesktop.org/software/appstream/docs/chap-Quickstart.html